packages/google_maps_flutter/google_maps_flutter_ios/ios/Classes/GoogleMapPolylineController.m

```diff
@@ -171,7 +191,7 @@ - (bool)hasPolylineWithIdentifier:(NSString *)identifier {
   }
   return self.polylineIdentifierToController[identifier] != nil;
 }
-+ (GMSMutablePath *)getPath:(NSDictionary *)polyline {
+- (GMSMutablePath *)pathForPolyline:(NSDictionary *)polyline {
   NSArray *pointArray = polyline[@"points"];
   NSArray<CLLocation *> *points = [FLTGoogleMapJSONConversions pointsFromLatLongs:pointArray];
```

Comment:
Actually, if this is just doing a JSON conversion, is there any reason not to just move this to FLTGoogleMapJSONConversions.m/.h, and eliminate the need to expose a private method at all?

Reply:
I don't see a response or change related to this comment.

Reply:
I just realised those comments never got posted. It said "Pending" and was stuck like that. Tripped up by github

<img width="791" alt="image" src="https://github.com/flutter/packages/assets/22373191/aa848278-1b37-4544-8337-ba02f639b4b0">

Reply:
> I'd assume the FLTJSONConversion file ought to be dumb and just convert and needn't know about the structure of the polyline object (`polyline[@"points"]`)

I'm not sure what this means; what would it mean for a JSON conversion method to not understand the structure of the object it is converting?

Reply:
As in this function calls through to a JSON conversion method (`pointsFromLatLongs`).

Other than that its all dealing with objects of maps, not untyped json fields. So I thought it makes more sense to be a method within the class. I have currently made it static again. Im not sure if this ought to be static/non-static but assumed you preferred it to be static based on above comments.

Reply:
> Other than that its all dealing with objects of maps, not untyped json fields.

Maps are a core JSON type, so I'm still not clear on what the distinction is here. But it's fine to leave it here for now.